test/LifToken.js

```diff
@@ -186,25 +186,24 @@ contract('LifToken', function(accounts) {
 
     let burned = web3.toWei(0.3);
 
+    assert.equal(accounts[0], await token.owner());
+
     // pause the token
     await token.pause({from: accounts[0]});
 
-    let thrown = false;
     try {
       await token.burn(burned, {from: accounts[5]});
-    } catch(e) {
-      thrown = true;
+    } catch (error) {
+      if (error.message.search('invalid opcode') == -1) throw error;
     }
-    assert.equal(true, thrown, "burn should have thrown because token was paused");
-
     await token.unpause({from: accounts[0]});
 
     // now burn should work
     await token.burn(burned, {from: accounts[5]});
 
     new BigNumber(initialBalance).minus(burned).
       should.be.bignumber.equal(await token.balanceOf(accounts[5]));
-    totalSupply0.minus(burned).should.be.bignumber.equal(await token.totalSupply.call());
+    totalSupply.minus(burned).should.be.bignumber.equal(await token.totalSupply.call());
```

Comment:
I liked `totalSupply0`, it was like "the initial total supply", but it's ok, we can keep this change :)